pkg/backends/newrelic/flush.go

```diff
@@ -142,12 +142,12 @@ func (f *flush) finish() {
 	}
 }
 
-func newMetricSet(n *Client, f *flush, metricName, Type string, Value float64, tags gostatsd.Tags, timestamp gostatsd.Nanotime) map[string]interface{} {
+func newMetricSet(n *Client, f *flush, metricName, Type string, Value float64, tags gostatsd.Tags) map[string]interface{} {
 	metricSet := map[string]interface{}{}
 
 	// GoStatsD provides the timestamp in Nanotime, New Relic requires seconds or milliseconds, see under "Limits and restricted characters"
 	// https://docs.newrelic.com/docs/insights/insights-data-sources/custom-data/send-custom-events-event-api#instrument
-	metricSet["timestamp"] = timestamp / 1e9
+	metricSet["timestamp"] = n.now().Unix()
```

Comment:
This will potentially cause a skew, as the metric data is built up. Because of second precision it's unlikely to be a problem, but I would still suggest avoiding it.  I believe the correct fix is to use `f.timeStamp`, from https://github.com/atlassian/gostatsd/blob/master/pkg/backends/newrelic/newrelic.go#L201

It's probably worth changing the type in the `flush` struct as well.

```suggestion
	metricSet["timestamp"] = int64(f.timestamp / 1e9)
```

(also in `newDimensionalMetricSet`)